Document/0x05b-Basic-Security_Testing.md

```diff
@@ -366,7 +366,109 @@ More infomation on using the Objection REPL can be found on the [Objection Wiki]
 
 ##### radare2
 
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1231>
+[radare2](https://rada.re/r/ "Radare2 official website") is a popular reverse engineering framework for analysis, disassembling, debugging and patching that is scriptable and supports many architectures and file formats including Android/iOS apps. For Android, Dalvik DEX (odex, multidex), ELF (executables, .so, ART) and Java classes are supported, and it even has a web server for collaborative work. It also contains several useful scripts that can help you during mobile application analysis as it offers low level disassembling and safe static analysis that comes handy when traditional tools fail.
+
+###### Installing radare2
+
+Please refer to [radare2's official installation instructions](https://github.com/radare/radare2/blob/master/README.md "radare2 installation instructions"). There are also installation instructions If you need to [install radare2 on Android](https://github.com/radare/radare2/blob/master/doc/android.md "Install radare2 on Android").
+
+###### Using radare2
+
+You can use `rafind2` to read strings directly from the encoded AndroidManifest.xml. Use this to list permissions, activities, providers, services, receivers and other information stored in AndroidManifest.xml
```

Comment:
you need to install axml2xml via r2pm, right?

Reply:
I didn't have to on an Ubuntu machine. Should I add this requirement?